tcms/testplans/templates/testplans/get.html

```diff
@@ -110,26 +112,51 @@ <h2 class="card-pf-title">
 <template id="test_case_row">
     <div class="list-group-item">
         <div class="list-group-item-header">
-            <div class="list-view-pf-expand">
-                <span class="fa fa-angle-right"></span>
-            </div>
-            <div class="list-view-pf-checkbox">
-                <input type="checkbox">
-            </div>
-            <div class="list-view-pf-actions">
-                <div class="dropdown pull-right dropdown-kebab-pf">
-                    <button class="btn btn-link dropdown-toggle" type="button" id="dropdownKebabRight9"
-                            data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
-                        <span class="fa fa-ellipsis-v"></span>
-                    </button>
-                    <ul class="dropdown-menu dropdown-menu-right" aria-labelledby="dropdownKebabRight9">
-                        <li><a href="#">{% trans 'Change status' %}</a></li>
-                    </ul>
-                </div>
-            </div>
             <div class="list-view-pf-main-info">
-                <!-- todo: this text is slightly misaligned. for some reason goes to the top -->
-                <a class="testcase" href="#"></a>
+                <div class="list-view-pf-left">
+                    <span class="fa fa-angle-right"></span>
+                </div>
+                <div class="list-view-pf-checkbox">
+                    <input type="checkbox">
+                </div>
+                <div class="list-view-pf-body">
+                    <div class="list-view-pf-actions">
+                        <div class="dropdown pull-right dropdown-kebab-pf">
+                            <button class="btn btn-link dropdown-toggle" type="button" id="dropdownKebabRight9"
+                                    data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
+                                <span class="fa fa-ellipsis-v"></span>
+                            </button>
+                            <ul class="dropdown-menu dropdown-menu-right" aria-labelledby="dropdownKebabRight9">
+                                <li><a href="#">{% trans 'Change status' %}</a></li>
+                            </ul>
+                        </div>
+                    </div>
+                    <div class="list-view-pf-main-info js-test-case-summary">
+                        <div class="list-view-pf-left">
+                            <a class="js-test-case-link"></a>
+                        </div>
+                        <div class="list-view-pf-body">
```

Comment:
This tag goes away and pf-additional-info becomes a direct sibling to pf-description. This will place the left-most icon somewhere in the middle of the page giving you much better vertical alignment.

Note: this will break the existing JS function because it assumes the elements are directly under summary. I suggest making the `getTestCaseRowContent()` function independent of layout - it can search directly under the current row and replace the values in-place (like it currently does).